src/codegen.cpp

```diff
@@ -5797,7 +5797,7 @@ extern "C" void jl_init_codegen(void)
             targetFeatures);
     assert(jl_TargetMachine && "Failed to select target machine -"
                                " Is the LLVM backend for this CPU enabled?");
-#if defined(USE_MCJIT) && !defined(_CPU_ARM_)
+#if defined(USE_MCJIT) && (!defined(_CPU_ARM_) && !defined(_CPU_PPC64_))
```

Comment:
I'm wonder what's wrong about fastisel... it works on AArch64 but not AArch32 for example......

What's the issue you see with FastISel?